NotebookExamples/polyglot/COVID-19.ipynb

```diff
@@ -148,110 +148,103 @@
     "            clearInterval(notebookScope.updateInterval);\n",
     "        }\n",
     "        function updateCovidPlot() {\n",
-    "            Promise.all([\n",
-    "                interactive.csharp.getVariable(\"deathsSeries\"),\n",
-    "                interactive.csharp.getVariable(\"confirmedSeries\"),\n",
-    "                interactive.csharp.getVariable(\"recoveredSeries\"),\n",
-    "                interactive.csharp.getVariable(\"currentSeries\"),\n",
-    "                interactive.csharp.getVariable(\"date\")])\n",
-    "                .then(data => {\n",
+    "            interactive.getVariables({ \"csharp\": [\"deathsSeries\", \"confirmedSeries\", \"recoveredSeries\", \"currentSeries\", \"date\"] })\n",
+    "                .then(variables => {\n",
     "                    if (typeof (document.getElementById(\"plotlyChartCovid\")) !== 'undefined') {\n",
+    "                        let csVariables = variables[\"csharp\"];\n",
     "                        Plotly.animate(\"plotlyChartCovid\", {\n",
     "                            data: [\n",
     "                                {\n",
-    "                                    lat: data[2].latitude,\n",
-    "                                    lon: data[2].longitude,\n",
-    "                                    text: data[2].data\n",
+    "                                    lat: csVariables[\"recoveredSeries\"].latitude,\n",
```

Comment:
Nit: More readable / understandable without the intermediate variable (`csVariables`) and with `csVariables.recoveredSeries` instead of `csVariables[\"recoveredSeries\"]`